modules/client-common/src/context-core.ts

```diff
@@ -90,7 +80,7 @@ export abstract class ContextCore {
       this.overriden.ipfsNodes = true;
     }
     // Set all the available addresses
-    for (const address of DeployedAddressesArray) {
+    for (const address of Object.values(ContractNames)) {
```

Comment:
Perhaps having a variable instead of `Object.values(ContractNames)` would be nice for better readability 
feel free to ignore